packages/ui/src/composables/index.ts

```diff
@@ -0,0 +1,43 @@
+export { useAccessTrap } from './useAccessTrap'
```

Comment:
```suggestion
export * from './useAccessTrap'
```
and the same below.
And we can avoid explicit type exports in this case.